bundler/lib/bundler/vendor/thor/lib/thor.rb

```diff
@@ -625,7 +625,7 @@ def normalize_command_name(meth) #:nodoc:
     # alias name.
     def find_command_possibilities(meth)
       len = meth.to_s.length
-      possibilities = all_commands.merge(map).keys.select { |n| meth == n[0, len] }.sort
+      possibilities = all_commands.reject { |_k, c| c.hidden? }.merge(map).keys.select { |n| meth == n[0, len] }.sort
```

Comment:
I had to add code to filter out hidden commands otherwise `install_or_cli_help` would show up in suggestons.